include/igl/ears.cpp

```diff
@@ -15,15 +15,15 @@ IGL_INLINE void igl::ears(
   Eigen::PlainObjectBase<Derivedear_opp> & ear_opp)
 {
   assert(F.cols() == 3 && "F should contain triangles");
-  Eigen::Array<bool,Eigen::Dynamic,3> B;
```

Comment:
I don't understand this change. `Array`s are fine...

Reply:
it created many problems, i dont remember exactly anymore which ones.

It was something with incompatibility with MatrixBase

Reply:
Probably some templating issues with the `Array<>` type being propagated to an function that expects a `MatrixBase<>` as input, which is not possible because of inheritance.